packages/deployment/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/deployment",
-  "version": "1.0.0-1.0.0-beta.9.0",
+  "version": "1.0.0-beta.10.0",
```

Comment:
this is done by lerna. You should run the `npm run update` command to have lerna update everything.